Rust generation broken

### Preflight checklist

- [X] I could not find a solution in the existing issues, docs, nor discussions.
- [X] I agree to follow this project's [Code of Conduct](https://github.com/ory/sdk/blob/master/CODE_OF_CONDUCT.md).
- [X] I have read and am following this repository's [Contribution Guidelines](https://github.com/ory/sdk/blob/master/CONTRIBUTING.md).
- [ ] This issue affects my [Ory Cloud](https://www.ory.sh/) project.
- [ ] I have joined the [Ory Community Slack](https://slack.ory.sh).
- [ ] I am signed up to the [Ory Security Patch Newsletter](https://ory.us10.list-manage.com/subscribe?u=ffb1a878e4ec6c0ed312a3480&id=f605a41b53).

### Describe the bug

The CI is currently failing because the Rust SDK is not compiling:



### Reproducing the bug

Run the CI or `scripts/generate.sh` followed by `scripts/test.sh`

### Relevant log output

```shell
error[E0412]: cannot find type `CourierMessageStatus` in this scope
   --> src/apis/v0alpha2_api.rs:787:145
    |
787 | ...tion<i64>, page: Option<i64>, status: Option<CourierMessageStatus>, recipient: Option<&str>) -> Result<Vec<crate::models::Message>, Er...
    |                                                 ^^^^^^^^^^^^^^^^^^^^ not found in this scope
    |
help: consider importing this enum
    |
12  | use crate::models::CourierMessageStatus;
    
``` 

(https://app.circleci.com/pipelines/github/ory/sdk/1201/workflows/01e1d2bd-ddce-469c-9625-be9fbd0ac24c/jobs/1111)


### Relevant configuration

_No response_

### Version

-

### On which operating system are you observing this issue?

Other

### In which environment are you deploying?

Other

### Additional Context

_No response_

--- a/openapi/templates/rust/reqwest/api.mustache
+++ b/openapi/templates/rust/reqwest/api.mustache
@@ -8,6 +8,8 @@ use reqwest;
 use crate::apis::ResponseContent;
 use super::{Error, configuration};
 
+// temporarely needed until https://github.com/OpenAPITools/openapi-generator/issues/5676 is fixed
+use crate::models::CourierMessageStatus;
 
 trait NumVecJoin {
     fn join(&self, sep: &str) -> String;

--- a/openapi/templates/rust/reqwest/api.mustache
+++ b/openapi/templates/rust/reqwest/api.mustache
@@ -8,9 +8,6 @@ use reqwest;
 use crate::apis::ResponseContent;
 use super::{Error, configuration};
 
-// temporarely needed until https://github.com/OpenAPITools/openapi-generator/issues/5676 is fixed
-use crate::models::CourierMessageStatus;
-
 trait NumVecJoin {
     fn join(&self, sep: &str) -> String;
 }
@@ -107,7 +104,7 @@ pub {{#supportAsync}}async {{/supportAsync}}fn {{{operationId}}}(configuration: 
 
 {{/vendorExtensions.x-group-parameters}}
 {{^vendorExtensions.x-group-parameters}}
-pub {{#supportAsync}}async {{/supportAsync}}fn {{{operationId}}}(configuration: &configuration::Configuration, {{#allParams}}{{{paramName}}}: {{^required}}Option<{{/required}}{{#required}}{{#isNullable}}Option<{{/isNullable}}{{/required}}{{#isString}}{{#isArray}}Vec<{{/isArray}}&str{{#isArray}}>{{/isArray}}{{/isString}}{{#isUuid}}{{#isArray}}Vec<{{/isArray}}&str{{#isArray}}>{{/isArray}}{{/isUuid}}{{^isString}}{{^isUuid}}{{^isPrimitiveType}}{{^isContainer}}{{#isBodyParam}}&crate::models::{{/isBodyParam}}{{/isContainer}}{{/isPrimitiveType}}{{{dataType}}}{{/isUuid}}{{/isString}}{{^required}}>{{/required}}{{#required}}{{#isNullable}}>{{/isNullable}}{{/required}}{{^-last}}, {{/-last}}{{/allParams}}) -> Result<{{#supportMultipleResponses}}ResponseContent<{{{operationIdCamelCase}}}Success>{{/supportMultipleResponses}}{{^supportMultipleResponses}}{{^returnType}}(){{/returnType}}{{#returnType}}{{{returnType}}}{{/returnType}}{{/supportMultipleResponses}}, Error<{{{operationIdCamelCase}}}Error>> {
+pub {{#supportAsync}}async {{/supportAsync}}fn {{{operationId}}}(configuration: &configuration::Configuration, {{#allParams}}{{{paramName}}}: {{^required}}Option<{{/required}}{{#required}}{{#isNullable}}Option<{{/isNullable}}{{/required}}{{#isString}}{{#isArray}}Vec<{{/isArray}}&str{{#isArray}}>{{/isArray}}{{/isString}}{{#isUuid}}{{#isArray}}Vec<{{/isArray}}&str{{#isArray}}>{{/isArray}}{{/isUuid}}{{^isString}}{{^isUuid}}{{#schema.dataType}}{{{schema.dataType}}}{{/schema.dataType}}{{^schema.dataType}}{{^isPrimitiveType}}{{^isContainer}}{{#isBodyParam}}&crate::models::{{/isBodyParam}}{{/isContainer}}{{/isPrimitiveType}}{{{dataType}}}{{/schema.dataType}}{{/isUuid}}{{/isString}}{{^required}}>{{/required}}{{#required}}{{#isNullable}}>{{/isNullable}}{{/required}}{{^-last}}, {{/-last}}{{/allParams}}) -> Result<{{#supportMultipleResponses}}ResponseContent<{{{operationIdCamelCase}}}Success>{{/supportMultipleResponses}}{{^supportMultipleResponses}}{{^returnType}}(){{/returnType}}{{#returnType}}{{{returnType}}}{{/returnType}}{{/supportMultipleResponses}}, Error<{{{operationIdCamelCase}}}Error>> {
 {{/vendorExtensions.x-group-parameters}}
 
     let local_var_client = &configuration.client;

